<!DOCTYPE html>
<html>
  <head>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
	<title>Maize Gametophyte Project Details</title>
	<link rel="stylesheet" href="/root.css" type="text/css">
  </head>
  <body>

	<div class="main">
	  
	  <!-- header -->
	  <div class="top_menu">
		<div class="title">Maize Gametophyte Project </div>
		<a href="/index.html">Home</a>
		<a href="/people.html">People</a>
		<a href="/aims.html">Goals</a>
		<span>Details</span>
		<a href="/fgb2/gbrowse">Genome Browser</a>
	  </div>
	  
	  <div class="main_content">

		<h1>Functional Genomics of Maize Gametophytes</h1>

		<p>
		  The female and male gametophytes make up the haploid phase of the angiosperm life cycle.  The female gametophyte is the egg producing embryo sac, and the male gametophyte is the sperm-producing pollen grain.  The use of mutants to reveal the various essential genes required for the development and functioning of both the male and female gametophytes has been hindered by the fact that the gametophytes are haploid.  As haploids with only one copy of each gene, the mutation of a gametophyte-essential gene will disrupt egg and/or pollen function, and consequently may not be transmitted to the next generation.  The inherent difficulty in identifying, recovering, and characterizing such mutations, makes it difficult to identify gametophyte-essential genes.
		</p>
		
		<p>
		  The NSF-funded project "<a href="http://www.nsf.gov/awardsearch/showAward.do?AwardNumber=0701731">Functional Genomics of Maize Gametophytes</a>" provides one approach to overcome this fundamental limitation.  This approach depends upon using genetic tools unique to the model crop Zea mays (maize):
		</p>
		
		<ol>
		  <li>
			<a href="#tertiary_trisomic_stocks">Tertiary trisomic stocks</a>, which possess an extra chromosome arm (plus centromere) in addition to the usual 10 maize chromosomes.
		  </li>
		  <li>
			<a href="#ac_transposons">The <i>Activator (Ac)</i> transposon</a>, which acts as a local mutagen when it jumps into neighboring genes.
		  </li>
		  <li>
			<a href="#seed_color_markers">Seed color markers</a> to detect the presence of the trisome, and to act as reporters for the <i>Ac</i>.
		  </li>
		</ol>

		<h2><a name="tertiary_trisomic_stocks">Tertiary Trisomic Stocks</a></h2> 

		<p>
		  Special lines of maize have been developed that contain the unusual B centromere in addition to a duplicate copy of one arm of each of the 10 maize chromosomes. Therefore, in diploid cells of such a maize line there are three copies of that particular chromosome arm: two arising from the autosomes and one from the single copy of the supernumerary duplicate arm, which we refer to as the “trisome” (although this is not a strictly defined term). The functional significance of trisomes arises during meiosis. During meiosis, each of the daughter cells will receive one of the autosomal chromosomes, but only half will receive the trisome. Why is this useful? Consider the situation in which a gametophyte-lethal mutation has occurred on the autosome. The gamete containing only the single autosome will fail to carry the mutation to the next generation. In contrast, the gamete containing both the autosome with the mutation and the trisome will function properly because the trisome contains a functional copy of the gene that was mutated on the autosome. Thus, tertiary trisomic stocks allow mutations in gametophyte-essential genes to be passed to the next generation for identification and study.
		</p>

		<h2><a name="ac_transposons">Ac Transposons</a></h2>  

		<p>
		  Transposons, or transposable elements, are DNA sequences that can move around (transpose) to different positions within the genome of a single cell.   <i>Activator (Ac)</i> transposons were one of the first transposable elements discovered by Barbara McClintock within the maize genome.  McClintock discovered that <i>Ac</i> transposons are self-sufficient in that they code for their own transposase enzyme and, therefore, can transpose independently.  When an <i>Ac</i> transposon “jumps” into another gene, its insertion disrupts the sequence of that gene, i.e. it is acting as a local mutagen.
		  Additional useful characteristics of the <i>Ac</i> transposon are:
		</p>

		<ol>
		  <li>
			<i>Ac</i> tends to transpose to linked sites (i.e. it jumps to sites in the DNA that are relatively close to the original site)
		  </li>
		  <li>
			<i>Ac</i> transposition frequently increases the number of <i>Ac</i> copies in the genome by leaving the donor locus intact and creating a new insertion
		  </li>
		  <li>
			Increasing dosage of <i>Ac</i> has a negative effect on excision that can be monitored with a <i>Dissociation (Ds)</i> element in a reporter gene (discussed <a href="#dissociator">below</a>).
		  </li>
		</ol>

		<p>
		  In this project, maize lines with <i>Ac</i> transposons in specific PCR-verified genomic locations have been crossed to tertiary trisomic lines.  In subsequent crosses, we will search for mutations created when the <i>Ac</i> jumps into genes essential for gametophyte development and function.
		</p>

		<h2><a name="seed_color_markers">Seed Color Markers</a></h2>  

		<p>
		  To screen for gametophyte lethal mutants in the tertiary trisomic stocks, we want to detect that the <i>Ac</i> transposon has jumped and whether or not the trisome is present.  A gametophyte-lethal mutation will be inferred by: 1) evidence that the <i>Ac</i> has jumped, and 2) segregation patterns that indicate the resultant mutation is only passed to progeny in the presence of the trisome.  Seed color markers will be used to detect the presence of the trisome and provide evidence of <i>Ac</i> transposition.
		</p>

		<p class="subheading"><b>Detection of the Trisomes</b></p>

		<p>
		  The maize lines have been designed so that there is a marker for each chromosome arm, with a recessive allele of the marker on the autosomes, and a dominant allele of the marker on the trisome. The presence of the single dominant allele on the trisome will be evident by a particular seed color characteristic as compared to the alternative color that occurs with the recessive allele. As examples, the following four chromosome arms being evaluated by the lab of <a href="http://www.science.oregonstate.edu/bpp/faculty/fowler/">John Fowler</a>, at Oregon State University:
		</p>


		<table class="experiment">
		  <tr class="experiment_head">
			<td style="width: 50px">
			  Chromosome Arm
			</td>
			<td>
			  Trisome (dominant allele)
			</td>
			<td>
			  Autosome (recessive allele)
			</td>
		  </tr>
		  <tr>
			<td>
			  9S
			</td>
			<td>
			  <div>
				<img src="/images/details/fig1/9SWx+.png" alt="Wx1 translucent kernel">
			  </div>
			  <i>Wx1</i>  translucent kernel 
			</td>
			<td>
			  <div>
				<img src="/images/details/fig1/9S wx1.png" alt="wx1 waxy kernel">
			  </div>
			  <i>wx1</i>  waxy kernel 
			</td>
		  </tr>
		  <tr>
			<td>
			  9L
			</td>
			<td>
			  <div>
				<img src="/images/details/fig1/9L Wc1.png" alt="Wc1 White cap (light yellow) kernel">
			  </div>
			  <i>Wc1</i>    White cap (light yellow) kernel 
			</td>
			<td>
			  <div>
				<img src="/images/details/fig1/9Lwc1.png" alt="wc1 standard yellow kernel">
			  </div>
			  <i>wc1</i>    standard yellow kernel 
			</td>
		  </tr>
		  <tr>
			<td>
			  1L
			</td>
			<td>
			  <div>
				<img src="/images/details/fig1/1LR1 standin.png" alt="R1 allows anthocyanin (purple) color">
			  </div>
			  <i>R1</i>     allows anthocyanin (purple) color 
			</td>
			<td>
			  <div>
				<img src="/images/details/fig1/1L r1 standin.png" alt="r1 no color (yellow)">
			  </div>
			  <i>r1</i>     no color (yellow) 
			</td>
		  </tr>
		  <tr>
			<td>
			  2S
			</td>
			<td>
			  <div>
				<img src="/images/details/fig1/2S B_standin.png" alt="B allows anthocyanin (purple) color">
			  </div>
			  <i>B</i>       allows anthocyanin (purple) color 
			</td>
			<td>
			  <div>
				<img src="/images/details/fig1/2S b standin.png" alt="b no color (yellow)">
			  </div>
			  <i>b</i>      no color (yellow) 
			</td>
		  </tr>
		</table>


		<p class="subheading"><b>Ac Markers</b></p>

		<a name="dissociator"></a>
		<p>
		  The detection of <i>Ac</i> transposons (and the increase in number of <i>Ac</i> copies in the genome after they jump) is accomplished by an additional seed color marker and the use of another type of transposon called a <i>Dissociator (Ds)</i> transposable element.  <i>Ds</i> transposons were also discovered by Barbara McClintock in the maize genome. McClintock discovered that <i>Ds</i> transposons are non-autonomous, as they are unable to transpose independently. <i>Ds</i> transposons do not code for their own transposase enzyme so they must depend on excess transposase produced by <i>Ac</i> transposons elsewhere in the genome. Therefore, <i>Ds</i> transposons remain immobile without the assistance of <i>Ac</i> transposons.   In the case of our project, the lines being evaluated have a <i>Ds</i> transposon inserted into a gene that affects seed color.  This gene can be anywhere in the genome.  When the <i>Ac</i> transposon is also present, the <i>Ds</i> transposon will occasionally jump out of its location in the seed color gene, allowing the seed color gene to regain its function and produce a detectable seed color phenotype; i.e., a patch or spot of purple color on an otherwise yellow- or bronze-colored seed.  In the case of the four chromosome arms being evaluated by the Fowler Lab, these so-called <i>Ds</i> “reporters” are:
		</p>

		<table class="experiment">
		  <tr class="experiment_head">
			<td style="width: 50px">
			  Chromosome Arm
			</td>
			<td>
			  Marker   (after <i>Ds</i> excises, due to presence of <i>Ac</i>)
			</td>
			<td>
			  Marker (<i>Ds</i> does not excise, no <i>Ac</i> present)
			</td>
		  </tr>
		  <tr>
			<td>
			  9S
			</td>
			<td>
			  <i>R1</i>     allows anthocyanin color
			</td>
			<td>
			  <i>r1</i>     no color (yellow)
			</td>
		  </tr>
		  <tr>
			<td>
			  9L
			</td>
			<td>
			  <i>R1</i>     allows anthocyanin color
			</td>
			<td>
			  <i>r1</i>     no color (yellow)
			</td>
		  </tr>
		  <tr>
			<td>
			  1L
			</td>
			<td>
			  <i>Bz</i>     purple color 
			</td>
			<td>
			  <i>bz</i>     bronze color    
			</td>
		  </tr>
		  <tr>
			<td>
			  2S
			</td>
			<td>
			  <i>Bz</i>     purple color
			</td>
			<td>
			  <i>bz</i>     bronze color
			</td>
		  </tr>
		</table>

		<table class="exp2">
		  <tr align="center">
			<td>
			  <img src="/images/details/fig2/fig2.png" alt="Spotted Corn Cob">
			</td>
		  </tr>
		  <tr>
			<td style="text-align: center">
			  <i>Ac</i> transposons in a kernel are detected by seed color markers. In the presence of <i>Ac</i> transposon, <i>Ds</i> transposable elements are induced to jump out of genes affecting seed color, resulting in kernels with specific color phenotypes, such as the spotting shown here.
			</td>
		  </tr>
		</table>

		<p>
		  Detecting <i>Ac</i> using the <i>Ds</i> seed color reporter is slightly more complicated than indicated by the discussion thus far. First, the <i>Ds</i> may jump at any time during the development of the seeds. If it jumps in a cell early in seed development (when there are few cells), then subsequent cell divisions will generate a large patch of cells in which the <i>Ds</i> is no longer in the seed color gene, and the seed will appear to have a large patch of the associated seed color phenotype. If the <i>Ds</i> jumps late in seed development (when most of the cells of the mature seed are already present), then subsequent cell divisions will be less numerous, resulting in only a few cells with the <i>Ds</i> removed from the seed color gene, and the seed will have only tiny spots of the associated seed color phenotype. So the size of the color spots provides an indication of when the <i>Ds</i> jumped during development.
		</p>

		<table class="exp2">
		  <tr>
			<td>
			</td>
			<td colspan="2" style="text-align:center; font-size: 18px">
			  Developmental time of <i>Ds</i> jump is indicated by kernel spotting pattern
			</td>
			<td>
			</td>
		  </tr>
		  <tr>
			<td class="image_line">
			  9S Trisome
			</td>
			<td colspan="2">
			  <img src="/images/details/fig3/9S.png" alt="9S Picture">
			</td>
		  </tr>
		  <tr>
			<td class="image_line">
			  1L Trisome
			</td>
			<td colspan="2">
			  <img src="/images/details/fig3/1L.png" alt="1L Picture">
			</td>
		  </tr>
		  <tr>
			<td>
			</td>
			<td>
			  Early Jump
			</td>
			<td align="right">
			  Late Jump
			</td>
		  </tr>
		</table>

		<p>
		  Furthermore, there is a negative dosage effect between the number of active <i>Ac</i> transposons and the incidence of <i>Ds</i> jumping.  When there are more copies of active <i>Ac</i>, there is both a decreased frequency and a delay in the excision of the <i>Ds</i> elements.  The observable result is fewer and smaller spots of the color phenotype.  If the <i>Ac</i> jumps and increases the number of <i>Ac</i> transposons in the genome from (for example) one copy to two copies, that jump will be detected by the presence of rare and small spots of the color marker, rather than common, large spots.
		</p>

		<h2>Steps in the Mutagenesis and Screening Process</h2>

		<p>
		  Given the tertiary trisomic stocks and the <i>Ac</i> lines, it still requires multiple generations to generate, assess, and identify lines with lethal gametophyte mutations. The following general steps are being used for this process:
		</p>

		<p class="subheading">
		  <b>1. Trisomic x <i>Ac</i> Cross</b>	
		</p>

		<p>
		  Plants that have a trisome, but no <i>Ac</i>, are crossed to plants that have an <i>Ac</i>, but no trisome. The location of the <i>Ac</i> is on the same chromosome arm that is ‘covered’ by the trisome. This cross produces seeds that have a trisome and 1 <i>Ac</i>, which can be recognized by their seed phenotype and planted for the next step.
		</p>

		<p class="subheading">
		  <b>2. Mutagenesis Cross (Trisome + 1 Ac)</b>
		</p>

		<p>
		  Plants that have one <i>Ac</i> plus a trisome for a given chromosome arm are grown from seeds selected from Cross 1.  <i>Ac</i> is active in these plants, and so it can transpose into a new location in the genome, potentially causing a gametophytic gene knockout.  These plants are crossed to a tester (a non-trisomic plant homozygous for the recessive allele of the trisome seed color marker, and for the <i>Ds</i> reporter).  The ears resulting from these outcrosses are screened to identify and select rare kernels with a new <i>Ac</i> by noting the presence of fine spots from the <i>Ds</i> reporter.  The new <i>Ac</i> is also a putative new mutant, which will be tested for a gametophytic effect in the next generation.
		</p>

		<p class="subheading">
		  <b>3. Test Cross (Trisome + 2 <i>Ac’s</i>)</b>
		</p>

		<p>
		  Finely spotted seeds with 2 <i>Ac</i>’s selected from the Mutagenesis Cross are grown and crossed again to the tester line. We examine the resultant ears to answer two questions:
		</p>

		<p>
		  a) <b>Did we pick the 2 <i>Ac</i> seeds correctly?</b> If so, the 2 <i>Ac</i>/fine spotting condition should now be visible on multiple kernels of the ear, as it is heritable.
		</p>

		<p>
		  b) <b>If we have heritable 2 <i>Ac</i>’s, is there reduced transmission of the <i>Ac</i>’s when the trisome is missing?</b>  To do this, we have to count all the kernel types on the ear, and look for ears where the fine spotting is under-represented in seeds lacking the trisome kernel marker.  <b>Such ears are likely to harbor new gametophytic mutants, and will be subjects of additional, more detailed tests.</b>
		</p>

		<p>
		  Our project is using this procedure to identify gametophytic mutants on <b>XXX</b> different chromosome arms in maize, with the combined efforts of scientists and students from a number of research institutions: the <a href="http://www.ciw.edu">Carnegie Institution for Science</a>, <a href="http://www.iastate.edu">Iowa State University</a>, <a href="http://www.oregonstate.edu/">Oregon State University</a>, <a href="http://www.sdstate.edu">South Dakota State University</a>, and <a href="http://www.stanford.edu">Stanford University</a>.  
		</p>

		<!-- footer -->
		<br clear="all"/>
		<div class="bottom">
		  <a href="http://nsf.gov/"><img src="/images/nsf1-small.png" alt="National Science Foundation"></a>
		  <a href="http://ciw.edu/"><img src="/images/cis-small.png" alt="Carnegie Institution of Science"></a>
		  <a href="http://iastate.edu/"><img src="/images/iowastate-stacked.png" alt="Iowa State University"></a>
		  <a href="http://osu.edu/"><img src="/images/ohiosu-stacked.png" alt="Ohio State University"/></a></td>
		  <a href="http://oregonstate.edu/"><img src="/images/osu-tag-small.png" alt="Oregon State University"></a>
		  <a href="http://sdstate.edu/"><img src="/images/sdsu-small.png" alt="San Diego State University"></a>
		  <a href="http://stanford.edu/"><img src="/images/stanford-small.png" alt="Stanford University"></a>
		  <a href="http://wlc.edu/"><img src="/images/wlc-small.png" alt="Wisconsin Lutheran College"></a>
		  <a href="http://cgrb.oregonstate.edu/"><img src="/images/cgrb-small-transparent.png" alt="Center for Genome Research &amp; Biocomputing"></a>
        </div>
		
	  </div>

	</div>
		
  </body>
</html>

